API.md

```diff
@@ -139,9 +147,14 @@ server.register(plugin, registerOptions, (err) => {
             if (err) {
                 throw err;
             }
-            server.start(() => {
-
-                console.log('hapi days!');
+            plugin = require('./awesome-plugin.js');
```

Comment:
The coding style does not conform to https://github.com/hapijs/contrib/blob/master/Style.md